qa/tasks/cephfs/filesystem.py

```diff
@@ -529,7 +530,12 @@ def __init__(self, ctx, fs_config={}, fscid=None, name=None, create=False):
             if fscid is not None:
                 raise RuntimeError("cannot specify fscid when creating fs")
             if create and not self.legacy_configured():
-                self.create()
+                self.create(recover=kwargs.pop("fs_recover", False),
+                            metadata_overlay=kwargs.pop("fs_metadata_overlay",
+                                                        False),
+                            fs_ops=kwargs.pop("fs_ops", None),
+                            yes_i_really_really_mean_it=kwargs.pop(
+                                "yes_i_really_really_mean_it", False))
```

Comment:
Since create too accepts kwargs, why not pass it directly?

Reply:
yup makes sense, passed it directly